src/vmware/azext_vmware/_help.py

```diff
@@ -277,7 +277,7 @@
     short-summary: Create a Site Recovery Manager (SRM) addon for a private cloud.
     examples:
     - name: Create a Site Recovery Manager (SRM) addon.
-      text: az vmware addon srm create --resource-group MyResourceGroup --private-cloud MyPrivateCloud --license-key "41915178-A8FF-4A4D-B683-6D735AF5E3F5"
+      text: az vmware addon srm create --resource-group MyResourceGroup --private-cloud MyPrivateCloud --license-key "41915-178A8-FF4A4-DB683-6D735"

```

Comment:
May I ask why do we need to modify the `--license-key`?

Reply:
It was the wrong format before. The test service now validates the format.

Reply:
OK